app/src/main/res/values-nb-rNO/strings_settings.xml

```diff
@@ -3,10 +3,10 @@
 
     <!-- Favorites -->
     <string name="button_select_favorites">Velg favoritter</string>
-    <string name="button_sort_az">Sort A-Z</string>
+    <string name="button_sort_az">Sorter A-Å</string>

```

Comment:
For this one, we will rather use "Sort alphabetically" because I noticed afterwards that "Sort A-Z" was not working with non-Latin alphabets (Cyrillic, Japanese).
Could you update it?